ProblemTwo/trivia.js

```diff
@@ -0,0 +1,20 @@
+document.addEventListener("DOMContentLoaded",()=>{
+  questionSelect(triviaData)
+})
+const triviaData =async()=>{
+    let url = "http://localhost:4001"
+    let response = await axios.get(url)
+    return response.data.results
+}
+
+
+const questionSelect = (data)=>{
+    let submitButton = document.querySelector("#submitButton")
+    let p = document.querySelector("#p")
+    let select =document.querySelector("#select")
+    let wrongOptions = data.incorrect_answers
+    let correct = data.correct_answer
+    p.innerText = data.question
+    submitButton.innerText = "Submit"
+    submitButton.addEventListener("click"(event))
```

Comment:
Missing a comma and the rest of the function here.

Something to notice: Typically you don't want to leave a function mid sentence before committing. 
Perhaps leave a comment on what you want to do.